src/components/Planning/GroupDialog/GroupProjectSearch.tsx

```diff
@@ -62,8 +64,8 @@ const GroupProjectSearch: FC<IProjectSearchProps> = ({
         (showAdvanceFields &&
           (!getValues('district')?.value ||
             (divisions.length > 0 && !getValues('division')?.value) ||
-            !getValues('subClass')?.value)) ||
-        (!showAdvanceFields && !getValues('subClass')?.value)
+            !getValues('subClass')?.value)) //||
+        //(!showAdvanceFields && !getValues('subClass')?.value)
```

Comment:
Is this line of code, that is commented out, still needed?

Reply:
I'll delete that.